### DISCUSSION TITLE
Add rootColor property to RingProgress

### DISCUSSION THREAD

**Discussion by Meierschlumpf** [mantinedev/mantine#3147] [Feature requests]
It would be nice if the color of the empty part in RingProgress would be changable. 
![image](https://user-images.githubusercontent.com/63781622/206485074-4b965880-c984-4c32-a3ca-e9e4300711f3.png)

I would create a pull request if you want.

**Comment by rtivital** [comment]
Sure, use `dev` branch as a base
